talent/offboarding.md

```diff
@@ -1,16 +1,16 @@
 This document describes the off-boarding process at source{d}.
 We believe that the off-boarding process should be smooth and clear in order to cause minimal disruption to every including member.
 
-The relationship with the company can finish in three different ways. 
+The relationship with the company can finish in two different ways. 
 
 #### The employee decides to leave
 
-- The employee has to tell the company (usually) 15 days in advance. 
-- The employee has to sign a resignation letter with their information and the end date. Example. 
+- The employee has to inform the company (usually) 15 days in advance. 
+- The employee has to sign a resignation letter with their information and the end date. [Example](https://drive.google.com/file/d/0B6AGEF9RyVDxdEFULWx1dzNSd0E/view?usp=sharing). 
```

Comment:
The employee has to sign a resignation letter with their information and the end date, as shown in the [example](https://drive.google.com/file/d/0B6AGEF9RyVDxdEFULWx1dzNSd0E/view?usp=sharing))